LARailsLearners/box-office
# Feature#11 add dot env gem

To test:
1. Add an example environment variable in `.env`. For example: `S3_BUCKET=YOURS3BUCKET`. 
2. In the `box-office.rb` file, after the `Dotenv.load` line, add the lines...
   
   `puts(
     ('YOURS3BUCKET' == ENV.fetch('S3_BUCKET')) ? 'pass' : 'FAIL'
   )`
3. Run `ruby box-office.rb`
4. You should see "pass" before the server displays its status.
5. Clean up. Undo steps 1 to 4.
